sensor_msgs_py/package.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>sensor_msgs_py</name>
+  <version>1.0.0</version>
```

Comment:
This needs to be the same version as the other packages already in this repository.  In this case, it looks like 2.1.0.